linux_os/guide/system/software/integrity/software-integrity/aide/aide_verify_acls/ansible/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_sle
```

Comment:
This could be OK unless it assumes specific package names.

Reply:
AFAIK aide is aide on multiple platforms including RHEL.